metadata-db-app/src/config/mdb_migrate_helper.clj

```diff
@@ -73,11 +79,18 @@
      (into (when (contains? (set concept-types) :access-group) ["cmr_groups"]))
      (into (when (contains? (set concept-types) :tag) ["cmr_tags"]))))))
 
+(def dummy-provider-for-small-prov
+  "Any schema changes that need to be made to provider specific tables in the system need to happen
+  to the small_prov tables. In an environment which does not have any small providers we need to
+  have a dummy small provider to ensure the tables have their schema updated."
+  {:small true :cmr-only true :provider-id "DUMMY" :short-name "DUMMY"})
+
 (defn get-collection-tablenames
   "Gets a list of all the collection tablenames. Primarily for enabling migrations of existing
   provider tables."
   []
-  (distinct (map #(concept-tables/get-table-name % :collection) (p/get-providers (config/db)))))
+  (let [providers (conj (p/get-providers (config/db)) dummy-provider-for-small-prov)]
```

Comment:
There's a bunch of places using these functions so I didn't refactor down to a single function. 

We add the dummy-prov because we want this function to always return all of the collection tables. SMALL_PROV_COLLECTIONS will exist regardless of whether or not there are any small providers, so we'll want to update its schema whenever we update other collection tables schemas. Similarly for granules.